spec/metanorma_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Metanorma do
   it "processes metanorma options inside Asciidoc" do
-    File.open("test.adoc", "w:UTF-8") { |f| f.write(ASCIIDOC_PREAMBLE_HDR) }
+    File.write("test.adoc", ASCIIDOC_PREAMBLE_HDR, encoding: "UTF-8")
```

Comment:
@CAMOBAP: Looks like we are doing this write and cleanup in each test, what do you think about extracting those to a separate method or something? And also maybe use a `temp` directory, so it doesn't clutter when we need run those test locally?

Reply:
Agree with @abunashir .

Reply:
Updated (in separate commit), @ronaldtse @abunashir could you please have a look https://github.com/metanorma/metanorma-cli/pull/148/commits/e04ddef3ffb363a41858da8ba4e22c977d93b94a